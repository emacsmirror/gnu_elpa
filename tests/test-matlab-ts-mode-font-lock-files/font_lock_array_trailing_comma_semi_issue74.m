% -*- matlab -*-

% https://stackoverflow.com/questions/32849720/why-is-a-trailing-comma-in-a-cell-array-valid-matlab-syntax
% See: https://github.com/acristoffers/tree-sitter-matlab/issues/74

c1 = {'foo',}

c2 = {{'foo',};;; {'bar',};;;}

m1 = [1,]

m2 = [1,2,3;;; 4,5,6;;;;]
