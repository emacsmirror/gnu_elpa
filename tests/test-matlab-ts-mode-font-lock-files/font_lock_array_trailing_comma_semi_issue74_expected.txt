C ccc cccccc ccc

C cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
C cccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

dd d dsssssdd

dd d ddsssssddddd dsssssdddddd

dd d dddd

dd d ddddddddd dddddddddd
